DolbyIO/Source/DolbyIOModule/Private/DolbyIO/Authenticator.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Dolby Laboratories
+
+#pragma once
+
+#include "Interfaces/IHttpRequest.h"
+
+class UDolbyIO;
+
+namespace DolbyIO
+{
+	class FAuthenticator final
+	{
+	public:
+		FAuthenticator(UDolbyIO&);
+
+		void GetToken(const FString& AppKey, const FString& AppSecret, int TokenExpirationTimeInSeconds);
```

Comment:
Add comment to state that this is only provided for convenience, for production, the developer needs to setup a backend service and reach to that service to get a token.